python/.flake8

```diff
@@ -1,6 +0,0 @@
-[flake8]
```

Comment:
We should not remove this file

Reply:
It's renamed into setup.cfg, where `isort` configuration also lives.